### PR TITLE
Updating qaHMPID task

### PR DESCRIPTION
**Pull request by gvolpe79**

### CODE REVIEW

**Comment by njacazio** [comment]
Ciao Giacomo, Megalinter is complaining, could you have a look?

**Comment by gvolpe79** [comment]
Fixed!

**Comment by njacazio** [comment]
Super, thx!

File: DPG/Tasks/AOTTrack/PID/qaHMPID.cxx [hunk 1]
```diff
@@ -30,26 +32,71 @@ struct pidHmpidQa {
   Configurable<float> maxDistance{"maxDistance", 5.f, "Maximum HMPID distance between the track and the cluster (cm)"};
   Configurable<float> minCharge{"minCharge", 120.f, "Minimum HMPID charge collected in the cluster"};
 
+  OutputObj<TTree> pTree{"pTree", OutputObjHandlingPolicy::AnalysisObject};
```
Comment (njacazio) [inline DPG/Tasks/AOTTrack/PID/qaHMPID.cxx:35]: Are you sure that you want to save a TTree? In this case I would suggest to produce tables instead, such that we can do the post analysis within the O2. @mfaggin what do you think?

Comment (mfaggin) [reply DPG/Tasks/AOTTrack/PID/qaHMPID.cxx:35]: I totally agree, better to go with a table.

Comment (njacazio) [reply DPG/Tasks/AOTTrack/PID/qaHMPID.cxx:35]: Perfect, @gvolpe79 let us know if you need some example to follow, the sintax is similar to the one of the tree but there's a better management (including e.g. merging) on the hyperloop.

Comment (gvolpe79) [reply DPG/Tasks/AOTTrack/PID/qaHMPID.cxx]: I removed the TTeee, could you merge like it is now? Then I will add the table!

**Review by njacazio — APPROVED** [review]
Thanks Giacomo

**Pull request merged**